website/source/docs/post-processors/amazon-import.html.md

```diff
@@ -185,6 +185,30 @@ using ovftool.
   ]
 ```
 
+## Amazon Permissions
+
+You'll need at least the following permissions in the policy for your IAM user
+in order to successfully upload an image via the amazon-import post-processor.
+``` json
+        "ec2:CancelConversionTask",
+        "ec2:CancelImportTask",
+        "ec2:CopyImage",
+        "ec2:CreateImage",
+        "ec2:CreateTags",
+        "ec2:DeregisterImage",
+        "ec2:DescribeConversionTasks",
+        "ec2:DescribeImageAttribute",
+        "ec2:DescribeImportImageTasks",
+        "ec2:DescribeImportSnapshotTasks",
+        "ec2:DescribeTags",
+        "ec2:ImportImage",
+        "ec2:ImportSnapshot",
+        "ec2:ImportVolume",
+        "ec2:ModifyImageAttribute",
+        "ec2:RegisterImage"
```

Comment:
Guessing here a bit, but I think these should be enough:

```json
  [
        "ec2:CancelImportTask",
        "ec2:CopyImage",
        "ec2:CreateTags",
        "ec2:DescribeImages",
        "ec2:DescribeImportImageTasks",
        "ec2:ImportImage",
        "ec2:ModifyImageAttribute"
  ]
```

Reply:
I'll switch them out and test.

Reply:
`"ec2:DeregisterImage"` Is also needed.